### DISCUSSION TITLE
Copycats on the Play Store

### DISCUSSION THREAD

**Discussion by Eggmanplant** [LawnchairLauncher/lawnchair#4864] [Q&A] [answered]
I've seen multiple copycats on the Play Store. Most of them have ads, and they don't follow the license. They don't give proper attribution, so this means they are illegal. I tried to report them, but I can't, because I'm not the copyright holder. Here are some:
[Pixel Launcher (not the Google app)](https://play.google.com/store/apps/details?id=app.pixel_launcher)
Seen others, hopefull they got taken down.
Will search for more.

**Comment by validcube** [comment, marked as answer]
Unfortunately, they are within their rights to modify and redistribute Lawnchair even if it is a closed source on the market.

See: https://www.apache.org/licenses/LICENSE-2.0

I've checked the launcher, and they do the following:
* Give attributions to Lawnchair (Settings > About)
* Don't say that Lawnchair endorse this product anyway
* The launcher does give notice of change in their change log

(The license must also be the same, but there's no way to check without viewing the source)

> [!CAUTION]
> Disclaimer: I'm not a lawyer but from the look of it, they don't seem to be violating the Apache-2.0 license

**Comment by Eggmanplant** [comment]
OK. Maybe other copycats I forgot violate the license.
Maybe [this (copycat Shade Launcher)](https://play.google.com/store/apps/details?id=dev.dworks.apps.alauncher.pro&hl=en-US) violates it. I can't check it, because it's paid, and unrelated to this thread, because it isn't Lawnchair.

**Comment by SuperDragonXD** [comment]
As stated by nullcube, we can't really do much regarding these. As such, closing this issue.
